csi/src/nodeplugin_grpc.rs

```diff
@@ -0,0 +1,85 @@
+//! The mayastor node plugin gRPC service
+//! This provides access to functionality that needs to be executed on the same
+//! node as a Mayastor CSI node plugin, but it is not possible to do so within
+//! the CSI framework. This service must be deployed on all nodes the the
```

Comment:
The additional "the" should be removed from the end of the line.

Reply:
Fixed.